src/site/layouts/va_form.drupal.liquid

```diff
@@ -62,15 +62,16 @@
           {% endif %}
         </dl>
 
-        <h2 class="vads-u-margin-top--4"">When to use this form</h3>
 
         {% if fieldVaFormUsage %}
+          <h2 class="vads-u-margin-top--4">When to use this form</h3>
           {{ fieldVaFormUsage.processed }}
-        {% endif %}
-
-        <h3>Downloadable PDF</h3>
 
-        <a href="{{ fieldVaFormUrl.uri }}" download>Download VA Form {{ fieldVaFormNumber }} (PDF)</a>
+          <h3>Downloadable PDF</h3>
```

Comment:
Hm so, I think we will have an issue with semantic headings if this remains an `h3` and `fieldVaFormUsage` is false

Reply:
Oh! Good catch, will update this! 🙂

Reply:
Wait, sorry I was confused lol So the `h3` tag is actually inside the `fieldVaFormUsage` conditional, so if `fieldVaFormUsage` is false then the h3 won't render, so we should be good here 🙂

Reply:
Ah okay, but I thought we would always want to show the download link for all forms, even if `fieldVaFormUsage` is null?

Reply:
We do! But I believe without the header, which is why I made the else condition here not have a `h` tag: https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/find-va-forms-redesign/form-detail-landing-pages/WIP-design-files/form-detail-FE-minimum-content-071620.pdf

Reply:
Not sure if that's desired but that's what I saw in the design spec per Jen's comment: https://github.com/department-of-veterans-affairs/va.gov-team/issues/11444#issuecomment-662739372

Reply:
Ahhhh I see now, thank you!